src/components/EntryForm.js

```diff
@@ -41,9 +51,13 @@ export default function EntryForm({ onCreateCard }) {
   function handleSubmit(event) {
     event.preventDefault();
     onCreateCard({
-      Date,
+      date: date.split(',').map(name => name.trim()),
+      time: time.split(',').map(name => name.trim()),
```

Comment:
Are you sure you need the "split by comma" method for date and time? usually a date would be typed in with only numbers and dots.
Unless you are asking for two different dates separated by commas

Reply:
I think I did that to test something. I can take them out by now. Good eyes 🕵️‍♂️